Lib/test/test_ttk_textonly.py

```diff
@@ -3,7 +3,7 @@
 # Skip this test if _tkinter does not exist.
 support.import_module('_tkinter')
 
-from tkinter.test import runtktests
+from test.tkinter_test import runtktests
 
 def test_main():
     support.run_unittest(
```

Comment:
After moving the tests, files like this (just contains an import and a wrapper to interface runtests with test_ttk) end up being a slightly confusing, mostly unneeded layer.

As long as many files are moved and history is broken, why not also clean up this tangle by merging the compat layer into the real test files?